cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/replication/BasicPullStrategy.java

```diff
@@ -76,26 +78,30 @@
      */
     private volatile boolean replicationTerminated = false;
 
-    public BasicPullStrategy(PullReplication pullReplication) {
-        this(pullReplication, null);
-    }
-
-    public BasicPullStrategy(PullReplication pullReplication,
-                             PullConfiguration config) {
+    public int changeLimitPerBatch = 1000;
 
-        Preconditions.checkNotNull(pullReplication, "PullReplication must not be null.");
+    public int batchLimitPerRun = 100;
 
-        if(config == null) {
-            config = new PullConfiguration();
-        }
+    public int insertBatchSize = 10;
 
-        this.config = config;
-        this.filter = pullReplication.filter;
+    public boolean pullAttachmentsInline = false;
 
-        CouchConfig couchConfig = pullReplication.getCouchConfig();
-        this.sourceDb = new CouchClientWrapper(couchConfig);
-        this.targetDb = new DatastoreWrapper((DatastoreExtended) pullReplication.target);
-        this.name = String.format("%s [%s]", LOG_TAG, pullReplication.getReplicatorName());
+    public BasicPullStrategy(URI source,
+                             Datastore target,
+                             PullFilter filter,
+                             List<HttpConnectionRequestInterceptor> requestInterceptors,
+                             List<HttpConnectionResponseInterceptor> responseInterceptors)
+    {
```

Comment:
Should this opening bracket be up on the previous line to match the rest of the method definitions?  As in:

```java
public BasicPullStrategy(URI source,
                         Datastore target,
                         PullFilter filter,
                         List<HttpConnectionRequestInterceptor> requestInterceptors,
                         List<HttpConnectionResponseInterceptor> responseInterceptors) {
...
```